lerna.json

```diff
@@ -2,5 +2,5 @@
   "packages": [
     "packages/*"
   ],
-  "version": "3.2.1"
+  "version": "3.2.2"
```

Comment:
Hmm, do you know why this version change is still showing up? Maybe you accidentally re-added the version changes?

Reply:
Hi @epicfaace
I updated the code for version changes. Thank you